src/portal/src/lib/components/config/gc/gc.component.html

```diff
@@ -1,15 +1,27 @@
-<form class="clr-form clr-form-horizontal">
-    <clr-checkbox-container>
-        <clr-checkbox-wrapper>
-            <input type="checkbox" clrCheckbox name="delete_untagged" id="delete_untagged"
-                   [(ngModel)]="shouldDeleteUntagged"/>
-            <label for="delete_untagged">{{'GC.DELETE_UNTAGGED' | translate}}</label>
-        </clr-checkbox-wrapper>
-    </clr-checkbox-container>
-</form>
 <div class="cron-selection">
-    <cron-selection [buttonMarginLeft]="'0'" [labelCurrent]="getLabelCurrent" #CronScheduleComponent [labelEdit]='getText'
+    <cron-selection [labelCurrent]="getLabelCurrent" #CronScheduleComponent [labelEdit]='getLabelCurrent'
                     [originCron]='originCron' (inputvalue)="scheduleGc($event)"></cron-selection>
-    <button class="btn btn-outline  gc-start-btn" (click)="gcNow()"
+</div>
+<div class="clr-row">
+    <div class="clr-col-2 flex-200"></div>
+    <div class="clr-col">
+        <span class="explain">{{'GC.EXPLAIN' | translate}}</span>
+    </div>
+</div>
+<div class="clr-row">
+    <div class="clr-col-2 flex-200"></div>
+    <div class="clr-col">
+        <clr-toggle-container class="mt-05">
+            <clr-toggle-wrapper>
+                <input type="checkbox" clrToggle name="delete_untagged" id="delete_untagged"
+                       [(ngModel)]="shouldDeleteUntagged"/>
+                <label class="font-weight-400" for="delete_untagged">{{'GC.DELETE_UNTAGGED' | translate}}</label>
+            </clr-toggle-wrapper>
+        </clr-toggle-container>
+    </div>
+</div>
+<div class="clr-row">
+    <button class="btn btn-primary  gc-start-btn" (click)="gcNow()"
```

Comment:
I found that there is a dry run button on the PM design, but there is no here, if this function is not good, the corresponding issue should not be closed

Reply:
For now, no try run funciton. Comfirmed with PM